static_content/_site/apply/thank-you/index.html

```diff
@@ -86,26 +86,62 @@ <h2 class="sr-only">Main actions</h2>
                 <span data-feather="chevron-down" aria-hidden="true"></span>
```

Comment:
this looks like the whole rendered HTML file - should it be included in this commit at all?

Reply:
It renders the symbol:
https://iconify.design/icon-sets/feather/chevron-down.html

Reply:
Yes, from https://feathericons.com/ -- it’s inlined SVG